recipe/0001-fix-version.patch

```diff
@@ -0,0 +1,10 @@
+--- pyproject.toml	2022-11-04 13:20:47.000000000 +0100
++++ pyproject.toml	2024-01-08 13:27:48.772803360 +0100
+@@ -1,6 +1,6 @@
+ [project]
+ name = "pydeck"
+-version = "0.8.0b4"
++version = "0.8.0"
```

Comment:
It was really weird change https://github.com/visgl/deck.gl/commit/9c04205cd03f693ee7ddd2b2c3d51930ce604f77
But we should patch also https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pydeck/_version.py

Reply:
Thanks for catching that! I have no idea how I missed it.

Reply:
oh actually on the packaged PyPi source that file has the correct version...that's why!